src/stylesheets/core/mixins/_icon.scss

```diff
@@ -108,7 +116,7 @@
     map-get($icon-object, "path"),
     $theme-image-path
   );
-  $ie11-variant: get-token-from-bg($background-color, $color-alt, "black");
+  $ie11-variant: get-token-from-bg($contrast-bg, $color-variant, "black");
```

Comment:
Shouldn't this be`$contrast-bg → $contrast-background` based on `background-color: color($theme-tooltip-background-color);` from the tooltip updates?

Reply:
I like to spell out `background` in theme settings, but I don't think we have to in mixins, functions, and internal vars. We could do a thorough pass on `bg` → `background` perhaps in another (later) PR.